explainer.md

```diff
@@ -575,6 +575,15 @@ const cookie = await cookieStore.get('session-id');
 console.log(cookie);
 ```
 
+### Modifying Insecure Cookies 
+The API will be able to fetch insecure cookies, however will only be able to modify secure cookies. This will mean that when modifying an insecure cookie with the API, the insecure cookie will automatically be modified to secure. 
```

Comment:
Phrasing suggestions:

"...however will..." &rarr; "...but will..."

"...modified to..." &rarr; "...changed to..."

Reply:
Sounds good!